frontend/src/index.tsx

```diff
@@ -48,9 +50,15 @@ const store = createStore(fakeReducer, applyMiddleware(backendMiddleware));
 
 backendMiddleware.run(backendSaga);
 
-ReactDOM.render(<Provider store={store}>
-  <App />
-</Provider>, document.getElementById('root'));
+ReactDOM.render(
+  <Provider store={store}>
+    <Router>
+      <Switch>
+        <Route path="/" component={PatientWelcome} />
+        <Route path="/admin" component={App} />
```

Comment:
No idea why, but when I navigate to /admin in the browser, I still see the PatientWelcome page. Can someone confirm?

Reply:
Can confirm.

Reply:
Need to switch the two Routes, then it works. `/` matches `/admin`, too.

Reply:
Changed this last minute. Thanks for finding the issue!